### PR TITLE
fix(scaffolder): properly post errors when calls fail

### PR DESCRIPTION
**Pull request by freben**
Some small tweaks to get things looking a bit better and reporting errors, while trying to reproduce #2364

### CODE REVIEW

File: plugins/scaffolder/src/api.ts [hunk 1]
```diff
@@ -49,17 +51,18 @@ export class ScaffolderApi {
     });
 
     if (response.status !== 201) {
-      throw new Error(await response.text());
+      const status = `${response.status} ${response.statusText}`;
+      const body = await response.text();
```
Comment (Rugvip) [inline plugins/scaffolder/src/api.ts:55]: I'd slap a `.trim()` on there

**Review by Rugvip — APPROVED** [review]

File: plugins/scaffolder/src/api.ts (hunk 1 above)
Comment (freben) [reply plugins/scaffolder/src/api.ts:55]: y tho, it's being slapped into the DOM right :)

Comment (Rugvip) [reply plugins/scaffolder/src/api.ts:55]: Because it's being templated into the error message

Comment (freben) [reply plugins/scaffolder/src/api.ts:55]: well ok then, added

**Pull request merged**